cocotb/share/makefiles/simulators/Makefile.aldec

```diff
@@ -90,7 +90,7 @@ ifneq ($(VHDL_SOURCES),)
 endif
 
 else ifeq ($(TOPLEVEL_LANG),vhdl)
-    GPI_ARGS = -loadvhpi $(LIB_DIR)/libcocotbvhpi_aldec:vhpi_startup_routines_bootstrap
+    GPI_ARGS = -loadvhpi \"$(LIB_DIR)/libcocotbvhpi_aldec:vhpi_startup_routines_bootstrap\"
```

Comment:
I don't fully udnerstand what's going on with these quotes, and why they need to be escaped - does aldec process these quotes itself?

Reply:
Same here: https://github.com/cocotb/cocotb/blob/master/cocotb/share/makefiles/simulators/Makefile.questa#L72